charts/core/traefik/questions.yaml

```diff
@@ -102,6 +102,14 @@ questions:
                 schema:
                   type: boolean
                   default: false
+              - variable: enableThemePark
+                label: "Enable theme.park"
+                description: Install plugin adding support for theme.park themes on supported apps.
+                  This is a 3rd party plugin!
+                  Please report any bugs to https://github.com/packruler/traefik-themepark
+                schema:
+                  type: boolean
+                  default: false
```

Comment:
The reason it is nested under `pilot` is because `pilot` is required for this to work. I agree the plugin section would be nice but I am not sure if there is a way to require other values without nesting.

Reply:
Ah, i see. Cool then

Reply:
That said, it may be possible to clone the plugin from GitHub Release via Kubernetes deploy process and then reference it as a local plugin. But I am not sure if that is worth the complexity...